untitled/src/homework2/task10.java

```diff
@@ -0,0 +1,24 @@
+package homework2;
+
+import java.util.Scanner;
+public class task10 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Please add positive number: ");
+        int a = sc.nextInt();
+
+        if (a <= 0) {
+            System.out.println("Number is negative or 0, try again ");
+        }
+        if (a > 0) {
```

Comment:
Could you test with number 9?
I am gettting output like:
_Not prime number: 9
 prime number: 9
Not prime number: 9
Not prime number: 9
Not prime number: 9
Not prime number: 9
Not prime numbe_r: 9
 prime number: 9_